From 60ec154ec67a18f2560651769eb51a101a50df98 Mon Sep 17 00:00:00 2001
From: Arulselvan M <arulselvan.m@intel.com>
Date: Fri, 8 Nov 2013 23:17:37 +0530
Subject: A new key for background music support over WIDI/HDMI

BZ: 150939

A new key for background music support over WIDI/HDMI is added in
AudioParameter to enable/disable this BGM based on settings
set in UI.

Category: Feature Differentiation
Domain: AUDIO_MEDIA-BGM
Origin: Internal
Upstream-Candidate: no, proprietary

Change-Id: I62e60ba541191632a51dde9647253c39f54b194d
Signed-off-by: Arulselvan M <arulselvan.m@intel.com>
Signed-off-by: johan.bohlin <johan.bohlin@intel.com>
---
 include/system/audio.h |    5 ++++-
 1 files changed, 4 insertions(+), 1 deletions(-)

diff --git a/include/system/audio.h b/include/system/audio.h
index 0f07644..ebc7012 100644
--- a/include/system/audio.h
+++ b/include/system/audio.h
@@ -368,6 +368,8 @@ enum {
                                AUDIO_DEVICE_IN_USB_DEVICE |
                                AUDIO_DEVICE_IN_DEFAULT),
     AUDIO_DEVICE_IN_ALL_SCO = AUDIO_DEVICE_IN_BLUETOOTH_SCO_HEADSET,
+    // device supporting remote background playback in addition to foreground playback
+    AUDIO_DEVICE_OUT_REMOTE_BGM_SINK = AUDIO_DEVICE_OUT_WIDI|AUDIO_DEVICE_OUT_REMOTE_SUBMIX,
 };
 
 typedef uint32_t audio_devices_t;
@@ -395,7 +397,8 @@ typedef enum {
     AUDIO_OUTPUT_FLAG_DEEP_BUFFER = 0x8, // use deep audio buffers
     AUDIO_OUTPUT_FLAG_COMPRESS_OFFLOAD = 0x10,  // offload playback of compressed
                                                 // streams to hardware codec
-    AUDIO_OUTPUT_FLAG_NON_BLOCKING = 0x20 // use non-blocking write
+    AUDIO_OUTPUT_FLAG_NON_BLOCKING = 0x20, // use non-blocking write
+    AUDIO_OUTPUT_FLAG_REMOTE_BGM = 0x40 // used for remote background playback
 } audio_output_flags_t;
 
 /* The audio input flags are analogous to audio output flags.
-- 
1.7.4.1

